src/components/Link/Link.scss

```diff
@@ -39,22 +44,9 @@
   @include ms-Link;
 }
 
-// TODO: Remove overrides below.
```

Comment:
It seems like you're removing all the content in BUTTON.ms-Link ?

Won't that break buttons that use ms-Link styling?